doc/content/verification_and_validation/ver-1kd.md

```diff
@@ -0,0 +1,124 @@
+\end{equation}
+
+\begin{equation}
+\frac{d C_{\text{T}_2}}{dt} = -K_1 c_{\text{H}_2} C_{\text{T}_2} + \frac{1}{2} K_2 C_{\text{HT}}^2
+\end{equation}
+
+where $K_1$ and $K_2$ represent the reaction rates for the forward and reverse reactions, respectively.
+
+At equilibrium, the time derivatives are zero
+
+\begin{equation}
+2K_1 C_{\text{H}_2} C_{\text{T}_2} - K_2 C_{\text{HT}}^2 = 0
+\end{equation}
+
+From this, we can derive the same equilibrium condition as used in TMAP7:
+
+\begin{equation}
+P_{\text{HT}} = \eta \sqrt{P_{\text{H}_2} P_{\text{T}_2}}
+\end{equation}
+
+where the equilibrium constant $\eta$ is defined as
+
+\begin{equation} \label{eq:eta}
+\eta = \sqrt{\frac{2K_1}{K_2}}
+\end{equation}
+
+Similarly to TMAP7, the equilibrium constant $\eta$ has been set to a fixed value of $\eta = 2$.
+
+The diffusion process for each species in the two enclosures can be expressed by
+
+\begin{equation}
+\frac{\partial C_1}{\partial t} = \nabla D \nabla C_1 + \frac{S}{V_1} kT,
+\end{equation}
+and
+\begin{equation}
+\frac{\partial C_2}{\partial t} = \nabla D \nabla C_2,
+\end{equation}
+
+where $C_1$ and $C_2$ represent the concentration fields in enclosures 1 and 2 respectively, $t$ is the time, $D$ denotes the diffusivity, $S$ the volumetric source rate, $V_1$ the volume of enclosure 1, $k$ the Boltzmann constant and $T$ the temperature.
+Note that the diffusivity may vary across different species and enclosures. However, in this case, it is assumed to be identical for all.
+
+The concentration in Enclosure 1 is related to the partial pressure and concentration in Enclosure 2 via the interface sorption law:
+
+\begin{equation}
+C_1 = K P_2^n = K \left( C_2 RT \right)^n
+\end{equation}
+
+where $R$ is the ideal gas constant in J/mol/K, $T$ is the temperature in K, $K$ is the solubility, and $n$ is the exponent of the sorption law. For Sieverts' law, $n=0.5$.
+
+## Results
+
+We assume that $K = 10/\sqrt{RT}$, which is expected to result in $C_1 = 10 \sqrt{C_2}$ at equilibrium.
+As illustrated in [ver-1kd_comparison_time_k10], since the chemical reactions occur immediately, an initial quantity of HT is present in Enclosure 1, while T$_2$ and H$_2$ have initially dropped to half their original amounts. The volumetric source contributes to an increase in T$_2$ within Enclosure 1, thereby enhancing its chemical reactions with H$_2$. Consequently, in Enclosure 1, H$_2$ pressure gradually decreases over time, while HT pressure rises. Similarly, the volumetric source increases T$_2$ pressure in Enclosure 2. In Enclosure 2, the increasing presence of T$_2$ leads to H$_2$ saturation, which in turn causes HT pressure to rise.
+
+Thus, it is crucial to ensure that the chemical equilibrium between HT, T$_2$ and H$_2$ is achieved. This can be verified in both enclosures by examining the ratio between $P_{\text{HT}}$ and $\sqrt{P_{\text{H}_2} P_{\text{T}_2}}$, which must equal $\eta=2$.
```

Comment:
```suggestion
For verification purposes, it is crucial to ensure that the chemical equilibrium between HT, T$_2$ and H$_2$ is achieved. This can be verified in both enclosures by examining the ratio between $P_{\text{HT}}$ and $\sqrt{P_{\text{H}_2} P_{\text{T}_2}}$, which must equal $\eta=2$.
```